grpc-bridge/src/main/java/org/jboss/resteasy/grpc/servlet/AsyncContextImpl.java

```diff
@@ -0,0 +1,101 @@
+package org.jboss.resteasy.grpc.servlet;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import jakarta.servlet.AsyncContext;
+import jakarta.servlet.AsyncListener;
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+
+public class AsyncContextImpl implements AsyncContext {
+
+   private static ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
IMO we should use the `ContextualExecutors.threadPool()` here so we can get the correct executor.

Reply:
Done